webapp/channels/src/actions/hosted_customer.tsx

```diff
@@ -182,7 +182,7 @@ export function retryFailedHostedCustomerFetches() {
     };
 }
 
-export function submitTrueUpReview(): ActionFunc {
```

Comment:
this automatically type inferred to NewActionFuncAsync now because of the changes in bindClientFunc, so this is awesome.

Reply:
Yeah, TS is really good at doing that. It doesn't have enough information to figure out actions that look like `() => (dispatch, getState) => ...`, so we have to either tell it the outer function's type or the types of `dispatch` and `getState`, so I went with just the outer `ActionFunc` type since it's just one instead of two.

I did consider leaving the explicit type definitions for this sort of action, especially for stuff in mattermost-redux, but I decided against it since the code requires less importing types this way so it looks cleaner